apiserver/facades/agent/machine/machiner_test.go

```diff
@@ -29,20 +30,26 @@ type machinerSuite struct {
 
 	resources *common.Resources
 	machiner  *machine.MachinerAPI
+
+	controllerConfigGetter *MockControllerConfigGetter
 }
 
 var _ = gc.Suite(&machinerSuite{})
 
 func (s *machinerSuite) SetUpTest(c *gc.C) {
 	s.commonSuite.SetUpTest(c)
 
+	ctrl := gomock.NewController(c)
+	s.controllerConfigGetter = NewMockControllerConfigGetter(ctrl)
```

Comment:
This doesn't look right.
- Mock controller and mock are instantiated in `SetUpTest`.
- There are no expectations against this Mock, but it passes tests where config should be retrieved.
- `commonSuite` embeds `ControllerSuite` (real Dqlite backing).